spec.bs

```diff
@@ -348,7 +348,11 @@ Each {{URLPattern}} object has an associated <dfn for=URLPattern>hash component<
 </div>
 
 <div algorithm>
-  To <dfn for=URLPattern>initialize</dfn> a {{URLPattern}} given a {{URLPattern}} |this|, {{URLPatternInput}} |input|, string or null |baseURL|, and {{URLPatternOptions}} |options|:
+  To <dfn for=URLPattern>initialize</dfn> a {{URLPattern}} given a {{URLPattern}} |this|, {{URLPatternInput}} |input|, string or null |baseURL|, and {{URLPatternOptions}} |options|, run [=create=] given |this|'s associated [=URL Pattern=], |input|, |baseURL|, and |options|.
```

Comment:
`associated [=URL Pattern=]` is not well defined. Once you define an actual field, this can be something like `[=URLPattern/URL pattern=]`.

However, an additional issue is that the associated URL pattern is never *set*; you just start modifying it.

It would be best if `[=create=]` did not accept a URL pattern as an argument, but instead created a new [=URL pattern=] and returned it.